src/routes/teams.js

```diff
@@ -764,11 +764,11 @@ async function deleteTeamMember(request, h) {
 
     if (!row) return Boom.notFound();
 
-    if (ROLES[row.dataValues.team_role] === 'owner' && !isAdmin) {
+    if (ROLES[row.dataValues.team_role] === 'owner') {
```

Comment:
I think this could be simplified as 
```suggestion
    if (row.team_role === ROLE_OWNER) {
```

Reply:
With the ORM fix, this suggestion can be applied :)